Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierBasicInformation.cshtml

```diff
@@ -176,7 +176,7 @@
                 <authorize scope="@OrgScopeRequirement.Editor">
```

Comment:
There's a bunch of other instances of link texts that don't make sense out of context as well - please could you address these at the same time?